### ISSUE TITLE
kv: trace RocksDB kv reads/writes 

### ISSUE THREAD

**Issue by arjunravinarayan** [arjunravinarayan/cockroach#15]
This is a Friday project.

- [ ] Build a tracing plugin that dumps all reads from RocksDB. We don't care about the values, but we do care about the keys and the size of the values scanned.
- [ ] Run a single node cockroach cluster with the tracer, run queries from TPC-H scalefactor 1, and see what KV queries are traced.
- [ ] Ping the traces over to @vijay03.

**Comment by vijay03** [comment]
Hey @arjunravinarayan, just following up. Did you get some time to work on this?

**Comment by arjunravinarayan** [comment]
Yes, but I didn't finish up in time. The tracing plugin design I originally thought of was at the wrong layer of abstraction. It was tracing things against the distributed sorted monolithic kv map, which is one level of abstraction too high, and not at the RocksDB level. Will try again.

**Comment by vijay03** [comment]
Got it. Thanks Arjun! I'll follow up next week.
